src/data/database.ts

```diff
@@ -1,215 +1,139 @@
-  Database.PROSITE,
-]);
-databaseCategoryToDatabases.set(DatabaseCategory.OTHER, [
-  // Not found: NextBio
-  Database.ChiTaRS,
-  Database.EvolutionaryTrace,
-  Database.GenomeRNAi,
-  Database.GeneWiki,
-  Database.PMAP_CutDB,
-  Database.PRO,
-]);
+export const databaseCategoryToString = {
+  [DatabaseCategory.CHEMISTRY]: 'Chemistry',
+  [DatabaseCategory.DOMAIN]: 'Family and domain databases',
+  [DatabaseCategory.EXPRESSION]: 'Gene expression databases',
+  [DatabaseCategory.FAMILY]: 'Protein family/group databases',
+  [DatabaseCategory.GEL]: '2D gel databases',
+  [DatabaseCategory.GENOME]: 'Genome annotation databases',
+  [DatabaseCategory.INTERACTION]: 'Protein-protein interaction databases',
+  [DatabaseCategory.ORGANISM]: 'Organism-specific databases',
+  [DatabaseCategory.OTHER]: 'Other',
+  [DatabaseCategory.PATHWAY]: 'Enzyme and pathway databases',
+  [DatabaseCategory.PHYLOGENOMIC]: 'Phylogenomic databases',
+  [DatabaseCategory.POLYMORPHISM]: 'Polymorphism and mutation databases',
+  [DatabaseCategory.PROTEOMIC]: 'Proteomic databases',
+  [DatabaseCategory.PROTOCOL]: 'Protocols and materials databases',
+  [DatabaseCategory.PTM]: 'PTM databases',
+  [DatabaseCategory.SEQUENCE]: 'Sequence databases',
+  [DatabaseCategory.STRUCTURE]: '3D structure databases',
+};
 
-export const entrySectionToDatabaseCategories = new Map<
-  EntrySection,
-  DatabaseCategory[]
->();
-entrySectionToDatabaseCategories.set(EntrySection.Expression, [
-  DatabaseCategory.ORGANISM,
-  DatabaseCategory.EXPRESSION,
-]);
-entrySectionToDatabaseCategories.set(EntrySection.FamilyAndDomains, [
-  DatabaseCategory.FAMILY,
-  DatabaseCategory.PHYLOGENOMIC,
-  DatabaseCategory.DOMAIN,
-]);
-entrySectionToDatabaseCategories.set(EntrySection.Function, [
-  DatabaseCategory.PATHWAY,
-  DatabaseCategory.FAMILY,
-]);
-entrySectionToDatabaseCategories.set(EntrySection.Interaction, [
-  DatabaseCategory.INTERACTION,
-  DatabaseCategory.CHEMISTRY,
-]);
-entrySectionToDatabaseCategories.set(EntrySection.NamesAndTaxonomy, [
-  DatabaseCategory.ORGANISM,
-]);
-entrySectionToDatabaseCategories.set(EntrySection.PathologyAndBioTech, [
-  DatabaseCategory.ORGANISM,
-  DatabaseCategory.OTHER,
-  DatabaseCategory.FAMILY,
-  DatabaseCategory.CHEMISTRY,
-  DatabaseCategory.POLYMORPHISM,
-]);
-entrySectionToDatabaseCategories.set(EntrySection.ProteinProcessing, [
-  DatabaseCategory.PROTEOMIC,
-  DatabaseCategory.GEL,
-  DatabaseCategory.PTM,
-  DatabaseCategory.OTHER,
-]);
-entrySectionToDatabaseCategories.set(EntrySection.Sequence, [
-  DatabaseCategory.SEQUENCE,
-  DatabaseCategory.GENOME,
-  DatabaseCategory.POLYMORPHISM,
-]);
-entrySectionToDatabaseCategories.set(EntrySection.Structure, [
-  DatabaseCategory.STRUCTURE,
-  DatabaseCategory.PROTOCOL,
-  DatabaseCategory.OTHER,
-]);
+export const {
+  databaseCategoryToNames,
+  databaseNameToCategory,
+  databaseToDatabaseInfo,
+} = getDatabaseInfoMaps(databaseInfo);
+
+const select = selectDatabases(databaseCategoryToNames);
```

Comment:
Would it be possible to add context to the name of the `select` variable? eg `databaseSelector`